Dockerfile

```diff
@@ -18,11 +18,34 @@ COPY pkg/ pkg/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# Base image
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+# Maintainer
+LABEL maintainer="Aerospike <support@aerospike.com>"
+
+# Labels
+LABEL name="aerospike-kubernetes-operator" \
+    vendor="Aerospike" \
+    version="2.0.0" \
```

Comment:
Can you pass this version from the makefile file as build args or replace using sed before the operator is built

Reply:
Done. Version will be taken from docker build arg